src/utils/gaa.js

```diff
@@ -259,7 +243,11 @@ body {
   display: none;
 }
 #${GOOGLE_SIGN_IN_BUTTON_ID} .abcRioButton.abcRioButtonBlue .abcRioButtonContents {
-  font-size: 15px !important;
+  font-size: 0 !important;
+}
+#${GOOGLE_SIGN_IN_BUTTON_ID} .abcRioButton.abcRioButtonBlue .abcRioButtonContents span[id^=not_signed_]:before {
```

Comment:
Could you add a comment about what this span does and why you need this class and the one above it?

Reply:
I added a couple CSS comments, please let me know what you think

Reply:
I think I get it now. If we have localized text, we set the default class font size to 0px and add this class with 15px font?

What most confuses me most is what `span[id^=not_signed_]` looks for and how the two classes work together. If you could add in some stuff about that in the comment that would be great.